textdb/textdb-common/src/main/java/edu/uci/ics/textdb/common/constants/SchemaConstants.java

```diff
@@ -15,9 +15,11 @@
     public static final String SPAN_END = "spanEnd";

```

Comment:
It may be better to define the members in the following order: fieldName, start, end, key, and value.  If you agree, please change the order accordingly.